package.json

```diff
@@ -27,7 +27,7 @@
     "jest": "jest --coverage",
     "build:tokens": "lerna run tokens --scope bpk-tokens",
     "build:svgs": "lerna run svgs --scope bpk-svgs",
-    "build": "npm run build:tokens && npm run build:svgs && lerna run build",
+    "build": "npm run build:tokens && npm run build:svgs && lerna run build && cd native/ && react-native link && cd -",
```

Comment:
surely `react-native link` only has to be run once?

Reply:
It should be run every-time a new font file is generated. If we add an icon, generate a new BpkIcon.ttf, and don't run `react-native link` then the new font will not be made available to our storybook app

Reply:
Ah ok - then:
```
cd native/ && react-native link && cd -
```
can be simplified to:
```
(cd native && react-native link)
```